notifications/v1/types.go

```diff
@@ -24,11 +25,19 @@ type UserDeletedData struct {
 }
 
 // UserRolesUpdatedData contains information of a "UserRolesUpdated" notification.
+// When a user's roles have been updated it is neccessary to query SAMS to get the
+// updated roles to determine if it was granted/revoked.
 type UserRolesUpdatedData struct {
```

Comment:
Can we also link to a Notion page here that we can keep up-to-date with best practices around handling role updates in general? It might be helpful to have a place where more in-depth documentation can live and be updated easily

Reply:
I created a stub notion page which I will add further documentation to 👍🏻 
https://www.notion.so/sourcegraph/SAMS-Roles-Resources-13ca8e11265880f9a573cac77070ca0c